.gitignore

```diff
@@ -14,7 +14,6 @@ workspace.code-workspace
 # ignoring the node_modules part, as it'd ignore every node_modules, and we have some for testing
 
 # Logs
-logs
```

Comment:
Removed this as it excluded the `logs` directory in Foundry test data. We can re-add it in subdirectories if it becomes necessary